app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -5,51 +5,56 @@ class AamvaStateMaintenanceWindow
     # All AAMVA maintenance windows are expressed in 'ET' (LG-14028),
     TZ = 'America/New_York'
 
+    Window = Data.define(
+      :cron,
+      :duration_minutes,
+    ).freeze
+
     MAINTENANCE_WINDOWS = {
       'AL' => [
         # First Monday of each month from 1 am – 7 am ET
-        { cron: '0 1 * * Mon#1', duration_minutes: 6 * 60 },
+        Window.new(cron: '0 1 * * Mon#1', duration_minutes: 6 * 60),
```

Comment:
Would this be an opportunity for `6.hours`?

Reply:
But then we'd have to change it to `duration:` and that's how we got into this mess in the first place. 😉 

I like it. I actually had a similar thought when I was reworking this, but didn't pursue it because I feared it was ballooning scope. Now that you mention it, I think it can be a very simple change.

Dealing with stuff like `3.5.hours` might make this a tiny bit clunky, but I don't think that's a blocker by any stretch.